### DISCUSSION TITLE
Not seeing data from slaves (heavy nodes)

### DISCUSSION THREAD

**Discussion by frustratedlinuxadmin** [Security-Onion-Solutions/securityonion#2045] [Unsupported Versions] [answered]
Hi!

Tried to setup Security onion, didn't work out that well. I have a single host sec-onion 2.3 that works flawlessly

But when i try to set it up in distributed mode im not seeing any data on the master (mgr) so-status gives all green statuses though.

I'm seeing some errors in filebeat logs. (DNS lookup failure, "seconion-heavy1" (this is the name of the heavy node) on "ip" no such host.

Now does sec-onion need the DNS server to work for it to forward logs? I can reach the machines, and they are both pingable as well as controllable from fleet manager.

Any ideas? 

Regards

**Comment by weslambert** [comment, marked as answer]
We should place a mapping in the hosts file, regardless of if DNS is available.  Is that mapping in place?  Did the IP change for the manager?  Did you use DHCP or static IP in setup for the manager?

**Reply by frustratedlinuxadmin** [reply]
Thanks for getting back to me! 

There is some mapping in the hosts file. On the Heavy node i have

127.0.0.1 seconion-heavy1 seconion-heavy1.localdomain localhost.localdomain.
::1 seconion-heavy1 seconion-heavy1.localdomain localhost
44.24.14.23 seconion-mgr

On the manager i have.
127.0.0.1 seconion-mgr secionion-mgr.localdomain localhost localhost.localdomain.
::1 seconion-mgr seconion-mgr.localdomain localhost

Shouldn't the managers hosts be populated with the heavy nodes as well?

It was setup with static IP's and no IP has changed :)

Thanks weslambert!

**Reply by frustratedlinuxadmin** [reply]
The IP of the heavy node is 44.24.14.24 btw

**Reply by TOoSmOotH** [reply]
on the manager do curl localhost:9200/_cluster/settings

**Reply by frustratedlinuxadmin** [reply]
Thanks! Figured it out myself! Didn't work when i used my built in DNS server.
Solved now :) Thanks!

**Reply by bicfordio** [reply]
@frustratedlinuxadmin can I inquire what your solution was? I am having a similar difficulty where after converting a standalone node to a heavy node and pairing with the manager, the manager node is unable to view any alerts coming from the heavy nodes within the dashboard.
